build/NuGetAdditionalFiles/Microsoft.Net.Compilers.props

```diff
@@ -11,15 +11,15 @@
        new properties in the local build task. -->
   <UsingTask TaskName="Microsoft.CodeAnalysis.BuildTasks.Csc"
              AssemblyFile="$(MSBuildThisFileDirectory)..\tools\Microsoft.Build.Tasks.CodeAnalysis.dll"
-             Condition="'$(MSBuildToolsVersion)' == '14.0'" />
+             Condition="'$(MSBuildToolsVersion)' == '14.0' OR '$(MSBuildToolsVersion)' == '15.0'" />
```

Comment:
Could we use condition that accepts all versions greater than 14? It feels like the current condition creates a maintenance burden.